tests/ast/data/DataFrame.to_pandas_batch.test

```diff
@@ -16,10 +16,16 @@ df = session.table("table1")
 
 df.to_pandas_batches()
 
```

Comment:
these are an artifact of the transitive closure/unparser, right?

Reply:
Yes, the Unparser isn't updated to only print statements whose `first_request_id` fields match the current `Request`'s ID field.